### ISSUE TITLE
Geyser Build #676 - Every Mappings/Customs Items/Armors/Blocks... Isn't working anymore | XBOX ONLY ISSUE

### ISSUE THREAD

**Issue by XPaladiumyX** [GeyserMC/Geyser#5044]
### Describe the bug

Hello, since the last update of Geyser and minecraft bedrock, (1.20.30) I had several reports from my players who said that custom things no longer appear and no longer work. I asked them all what platforms they were on and they all answered Xbox. From what I understood from our tests on xbox, when the xbox player joins the server it will ask to download the texture pack etc... But after finishing the installation, the pack is not applied in the game and when leaving the game the packs are not displayed in the minecraft data cache.

I did the same test on PC, mobile and playstation, and there are no problems like this. It seems to only do this on xbox players.

**THIS ISSUE HAS ONLY BEEN TESTED ON GEYSER VELOCITY Build #676**

### To Reproduce

1. Start your Xbox (where the problem was most encountered: xbox series S/X)
2. Join a server with a custom pack using GeyserMC on 1.20.30.
3. Download the pack.
4. The packs seems to not be applied every custom things are invisible or not showing.
5. Leave the server.
6. Go to your minecraft setting -> Storage -> Data Cache Storage.
7. Try to find the pack you downloaded previously on the server.
8. The pack does not appear in the cache.

### Expected behaviour

That it should works on xbox and that thet pack should be displayed knowing that you downloaded it before.

### Screenshots / Videos

XBOX JOINING IMAGES
1. Joinning the server, ur getting asked to download the pack.

![rn_image_picker_lib_temp_e6d0d611-5fe8-48b8-9f9f-cf5e88f383bd](https://github.com/user-attachments/assets/522be510-3664-4a75-a10d-5140dbc2e201)

2. Your downloading the pack
![rn_image_picker_lib_temp_a63e2824-4d5e-4260-8743-fbb8e54674c2](https://github.com/user-attachments/assets/c7e3fbdb-dcfa-44ff-b92b-4ec4eba6cb96)

3. In game, every custom images/items/anything isnt showing.
![rn_image_picker_lib_temp_a00204ee-5d38-4636-a4c2-deadd8847f68](https://github.com/user-attachments/assets/bf6eb5de-790c-44a7-8f8f-be522fe99f7d)

### Server Version and Plugins

Server Version : 1.20.1 version git-Paper-196 (MC: 1.20.1) (Implementing API version 1.20.1-R0.1-SNAPSHOT) (Git: 773dd72)
Geyser Velocity Version : Latest Build #676 2.4.3-b676 (git-master-7332c6b) (Java: 1.21, Bedrock: 1.20.80/1.20.81 - 1.21.30)
Floodgate Velocity Version : Latest Build #111 2.2.3-SNAPSHOT (b111-3db8e59) (branch: master)
Floodgate Spigot/Paper (on backend servers) : Latest Build #111 2.2.3-SNAPSHOT (b111-3db8e59) (branch: master)

### Geyser Dump

https://dump.geysermc.org/Vr76NUXyxt9N5Ms4AczVpm8uUnKdIwJK

### Geyser Version

2.4.3-b676 (git-master-7332c6b)

### Minecraft: Bedrock Edition Device/Version

1.20.30 XBOX SERIES S/X

**Comment by onebeastchris** [comment]
Does custom items/armor etc show up when playing on featured servers?

**Comment by XPaladiumyX** [comment]
> Does custom items/armor etc show up when playing on featured servers?

About featured server, i didn't tested myself as i don't have an xbox sadly. But i asked the peoples who's having this issue on xbox, and apparently they said that it was working on featured server and that they wasn't encountering this issue, as i know (i'm not really sure) the featured servers are not using GeyserMC right? If it is right then i think it comes from the GeyserUpdate or Minecraft i don't really know because i just discovered this issue today and i'm not at my home to make more tests in details.

**I also forgot to say that these tests were done with MCXBOXBROADCAST from Rtm and that joining via bedrockconnect or dns was not tested.**

**Comment by onebeastchris** [comment]
Is this still an issue? We've heard no other reports of this, so i'm inclined to believe that it's not caused by a bug within Geyser

**Comment by XPaladiumyX** [comment]
I don't think so, i am still having the issue but i really don't know about this so yeah you can close it its useless. Thank you for your support.
